kernel/src/main/scala/cats/kernel/Order.scala

```diff
@@ -139,7 +139,16 @@ abstract class OrderFunctions[O[T] <: Order[T]] extends PartialOrderFunctions[O]
     ev.max(x, y)
 }
 
-object Order extends OrderFunctions[Order] {
+trait OrderToOrderingConversion {
+  /**
+   * Implicitly derive a `scala.math.Ordering[A]` from a `Order[A]`
+   * instance.
+   */
+  implicit final def catsKernelOrderingForOrder[A](implicit ev: Order[A]): Ordering[A] =
+    ev.toOrdering
+}
+
+object Order extends OrderFunctions[Order] with OrderToOrderingConversion {
```

Comment:
Now I don't know why I thought that would be a valid point to make. It has to be a trait for `cats.implicits._`. Sorry about the noise.